api/commons/parameters/autoApplyTags.yaml

```diff
@@ -1,8 +1,8 @@
 in: query
 name: "autoApplyTags"
 description: >
-  Optionally update resource tags on the fly without performing a roll.
+  Optionally update resource tags immediately without performing a roll.
 schema:
   type: boolean
   example: true
-  default: false
```

Comment:
bad way of PR

Reply:
HI @EladTanaami  Why is this a bad way of PR. How would you like us to do this?